superset-frontend/spec/javascripts/dashboard/components/gridComponents/Tabs_spec.jsx

```diff
@@ -113,7 +108,7 @@ describe('Tabs', () => {
     const createComponent = sinon.spy();
     const wrapper = setup({ editMode: true, createComponent });
     wrapper
-      .find('.dashboard-component-tabs .nav-tabs a')
+      .find('.dashboard-component-tabs .ant-tabs-nav-add')
```

Comment:
might be worth considering a data-test attribute here, in case our class names change for library/styling purposes.